visualdl/storage/CMakeLists.txt

```diff
@@ -14,3 +14,5 @@ add_dependencies(entry storage_proto im)
 add_dependencies(record storage_proto entry)
 add_dependencies(tablet storage_proto)
 add_dependencies(storage storage_proto record binary_record tablet entry)
+
+target_link_libraries(storage storage_proto record binary_record tablet entry)
```

Comment:
the pre-commit should be executed, that will help to reformat the code style and some delete such empty lines.

run pre-commit by `pre-commit run -a`, need to install first.

Reply:
Sorry for not reading the develop guide carefully. Coding style fixed.